web/models/user.ex

```diff
@@ -7,6 +7,13 @@ defmodule CodeCorps.User do
     field :password, :string, virtual: true
     field :encrypted_password, :string
 
+    field :first_name, :string
+    field :last_name, :string
+
+    field :twitter, :string
+    field :biography, :string
+    field :website, :string
+
```

Comment:
Any reason we're breaking up the fields into groups? Also may consider alpha ordering.